auxiliary/conclusion.tex

```diff
@@ -60,6 +60,7 @@
 	\begin{figure}
 		\centering
 		\includegraphics[width=1.5\linewidth]{diagrams/Conclusion}
+		\caption{Research data work outputs}
```

Comment:
what about, 'development research data outputs'? just to make this more aligned with the book title